### ISSUE TITLE
IBL TODO: Rerun BWM conversion

### ISSUE THREAD

**Issue by CodyCBakerPhD** [catalystneuro/IBL-to-nwb#62]
Several hundred new sessions were added

All existing sessions must be re-added

**Comment by h-mayorquin** [comment]
The full BWM re-conversion has been completed. All sessions have been re-converted with the updated pipeline and uploaded to DANDI (dandiset 000409). The conversion infrastructure, including EC2-based parallel processing and DANDI upload orchestration, is part of the codebase in PR #108.

**Comment by CodyCBakerPhD** [comment]
Coolio!

The massive AI dump of #108 is hard to parse - did you end up using any of the NeuroConv AWS tooling or containres to help with the EC2?

Did you use any spot pricing? (just curious personally)

**Comment by h-mayorquin** [comment]
No spot pricing and the aws infrastructure used in this project was ad-hoc; not the one in neuroconv.
